src/shared/Core/UriExtensions.cs

```diff
@@ -88,12 +88,14 @@ public static IEnumerable<string> GetGitConfigurationScopes(this Uri uri)
 
             string schemeAndDelim = $"{uri.Scheme}{Uri.SchemeDelimiter}";
             string host = uri.Host.TrimEnd('/');
+            // If port is default, don't append
+            string port = uri.Port == 80 ? "" : $":{uri.Port}";
```

Comment:
The default port number depends on the scheme/protocol (for `https` it's 443).

You can use the [`System.Uri::IsDefaultPort`](https://learn.microsoft.com/en-us/dotnet/api/system.uri.isdefaultport?view=net-6.0) instance property to check this easily however!